Dockerfile

```diff
@@ -4,8 +4,6 @@ FROM golang:$GO_BASE_VERSION as builder
 ARG TARGETOS
 ARG TARGETARCH
 
-# TODO remove this line once the repo is public
-ENV GOPROXY "https://goproxy.build.ue1.snooguts.net"
 ENV GOPRIVATE ""
 ENV GONOSUMDB "github.com/*"
```

Comment:
You'll want to remove all of the proxy-related configuration, not just GOPROXY.

Reply:
Removed the whole proxy paragraph